src/pybind/mgr/cephadm/module.py

```diff
@@ -219,92 +219,92 @@ class CephadmOrchestrator(orchestrator.Orchestrator, MgrModule,
         ),
         Option(
             'container_image_prometheus',
```

Comment:
These names like `container_image_prometheus` should ideally get populated from python-common/ceph/cephadm/images.py at this point. Maybe we should modify `DefaultImages` to only include the part of it specific to the daemon type (so `container_image_prometheus` for this one) and have callers handle prepending the `mgr/cephadm/` to the front. That will get us to the point where both the config name and the image are both coming from one spot. Currently you'd still have to specify the config option name both in images.py and here. Additionally, there is an idea to have these options, as in the entirety of
```
Option(
      'container_image_prometheus',
      default=DefaultImages.PROMETHEUS.value.name,
      desc='Prometheus container image',
)
```
be generated dynamically based on the contents of images.py so when you add a new image you only need to touch that one file. I don't think that portion needs to be in this PR, but what I was mentioning earlier about pulling `container_image_prometheus` from images.py rather than hardcoding it would be a step in that direction.

Reply:
Updated the PR